package.json

```diff
@@ -23,7 +23,7 @@
     "broccoli-merge-trees": "^1.0.0",
     "ember-cli-babel": "^6.6.0",
     "es5-shim": "~4.5.2",
-    "pixi.js": "~3.0.9"
+    "pixi.js": "~4.5.2"
```

Comment:
can you guarantee that https://github.com/simplabs/ember-cli-pixijs/blob/master/index.js#L36 will still work with all future versions of `pixi.js`?